src/testing/testing_doc.nit

```diff
@@ -293,6 +299,44 @@ class NitUnitExecutor
 		var res = toolcontext.safe_exec(cmd)
 		return res
 	end
+
+	# Compile an unit file and return the compiler return code
+	#
+	# Can terminate the program if the compiler is not found
+	private fun compile_single_docunits(dus: Array[DocUnit]): Int
+	do
+		# Generate all unitfiles
+		var files = new Array[String]
+		for du in dus do
+			files.add generate_single_docunit(du)
+		end
+
+		if files.is_empty then return 0
+
+		toolcontext.info("Compile {dus.length} single(s) doc-unit(s) at once", 1)
+
+		# Mass compile them
+		var nitc = toolcontext.find_nitc
+		var opts = new Array[String]
+		if mmodule != null then
+			opts.add "-I {mmodule.filepath.dirname}"
```

Comment:
Are you sure it works when the module is inside a group? Example: `cd contrib/foo && nitunit foo::bar::baz`